### PR TITLE
BINF-540: rna support column

### PR DESCRIPTION
**Pull request by kmhernan**
* Added `column_value` enum `YesNoOrUnknownEnum`
* Added `column_type` `YesNoOrUnknown`
* Created unit test for new column type/value
* Created `gdc-2.0.0-aliquot-merged` schema
* Created `gdc-2.0.0-aliquot-merged-masked` schema
* Both schemas include `RNA_Support` column as column type `YesNoOrUnknown`

### CODE REVIEW

File: maflib/column_values.py [hunk 1]
```diff
@@ -32,6 +32,15 @@ class NullableYOrNEnum(MafEnum):
     Yes = "N"
```
Comment (czyszCTDS) [inline maflib/column_values.py:32]: Just noticed..... 🙃

File: maflib/schemas/gdc-2.0.0-aliquot-merged.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
+{
+	"version" : "gdc-1.0.0",
+	"annotation-spec" : "gdc-2.0.0-aliquot-merged",
+	"extends" : "gdc-1.0.0",
```
Comment (czyszCTDS) [inline maflib/schemas/gdc-2.0.0-aliquot-merged.json:4]: Shouldn't this technically extend `gdc-1.0.0-aliquot-merged` with the appropriate columns `filtered`?

File: maflib/column_values.py (hunk 1 above)
Comment (kmhernan) [reply maflib/column_values.py:32]: oh nice find, will fix this before merging

File: maflib/schemas/gdc-2.0.0-aliquot-merged.json (hunk 2 above)
Comment (kmhernan) [reply maflib/schemas/gdc-2.0.0-aliquot-merged.json:4]: yup, just lazy

File: maflib/column_values.py (hunk 1 above)
Comment (kmhernan) [reply maflib/column_values.py]: Changed this and added in unit tests

File: maflib/schemas/gdc-2.0.0-aliquot-merged.json (hunk 2 above)
Comment (kmhernan) [reply maflib/schemas/gdc-2.0.0-aliquot-merged.json]: I stopped being lazy and did the inheritance

**Review by czyszCTDS — APPROVED** [review]

**Pull request merged**